### DISCUSSION TITLE
how to use  ( Write prompts to file  )

### DISCUSSION THREAD

**Discussion by CHETHAN562** [adieyal/sd-dynamic-prompts#273] [Q&A]

**Comment by adieyal** [comment]
Can you elaborate? You simply check the box and it will create a file in the output folder with all your prompts. You can pair it will don't generate images if you simply want to generate prompts. (Note - don't generate images will still generate one image as auto1111 expects this)

**Reply by ImStarWars** [reply]
Not sure what the OP was asking but my question falls under this "how to use" banner so I apologize for the necro on this. Lemme know if I should spin up a separate one.

What I've noticed is that when I first started using this extension about 6 or so months ago, I'd generate 10 different images and I'd get one txt file per image with the prompt/meta data for that image as I have set in my vanilla A1111 settings, and then I'd get one file from this extension with ALL the prompt as I wrote it at the top and then each of the combinatorial prompts below in the order they were doing to be generated. However lately I've been getting that same txt file from A1111 and then instead of one file with everything from the extension, I get one file per image (regardless of if I do them as multiple images in one batch or multiple batches of one image) with the prompt as written at the top and the combinatorial version of the prompt for that specific image below.

Was something adjusted in an update to this extension or did one of my settings possibly get flipped during at A1111 update? Not the end of the world, but I liked how it did it before since it felt cleaner so if I could get it back to that, that'd be nice.
